helpers/add-listener-to-one-or-many.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+var toObject = require('../utils/to-object')
+
+module.exports = addListenerToOneOrMany
+
+function addListenerToOneOrMany (state, eventName, one, handler) {
+  if (!state.pouchDBChangesFeedEmitter) {
+    state.pouchDBChangesFeedEmitter = this.changes({
+      since: 'now',
+      live: true,
+      include_docs: true
+    })
+    .on('create', function (change) {
+      state.emitter.emit('add', toObject(change.doc))
+      state.emitter.emit('change', 'add', toObject(change.doc))
```

Comment:
np, I like that style more too, but some people thinks it's overkill so I've leave it out this time :).